### PR TITLE
3 two pointers questions solved with tests

### PR DESCRIPTION
**Pull request by shree-ankush**
Problem : Time Taken
SquaresOfASortedArray : 25min
CountPairsWhoseSumIsLessThanTarget : 19min
FindFirstPallindromicString : 1hr (HELPED)

### CODE REVIEW

File: src/main/java/org/akv/two_pointers/CountPairsWhoseSumIsThanTarget.java [hunk 1]
```diff
@@ -41,9 +41,10 @@ public int countPairsWithLoop(int [] arr, int target){
         for (int i =0;i<arr.length ;i++ ) {
             for (int j = i+1 ;j<arr.length ;j++ ) {
                 if(arr[i]+arr[j] < target){
-                    System.out.println(arr[i]+" "+arr[j]);
+//                    System.out.println(arr[i]+" "+arr[j]);
```
Comment (PPapnai) [inline src/main/java/org/akv/two_pointers/CountPairsWhoseSumIsThanTarget.java:44]: Never leave commented code in your file.

Comment (shree-ankush) [reply src/main/java/org/akv/two_pointers/CountPairsWhoseSumIsThanTarget.java:44]: Will keep in mind

File: pom.xml [hunk 2]
```diff
@@ -13,5 +13,61 @@
         <maven.compiler.target>17</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
```
Comment (PPapnai) [inline pom.xml:18]: why are we adding this dependency so many times. Did it refused to get included in the first time itself 😆

File: src/main/java/org/akv/two_pointers/easy/CountPairsWhoseSumIsThanTarget.java [hunk 3]
```diff
@@ -41,9 +41,9 @@ public int countPairsWithLoop(int [] arr, int target){
         for (int i =0;i<arr.length ;i++ ) {
             for (int j = i+1 ;j<arr.length ;j++ ) {
                 if(arr[i]+arr[j] < target){
-                    System.out.println(arr[i]+" "+arr[j]);
                     countPairs++;
                 }
+//                else System.out.println("in else");
```
Comment (PPapnai) [inline src/main/java/org/akv/two_pointers/easy/CountPairsWhoseSumIsThanTarget.java:46]: commented code!

File: src/main/java/org/akv/Main.java [hunk 4]
```diff
@@ -1,36 +1,39 @@
...
-import org.akv.two_pointers.SquaresOfASortedArray;
-
-import java.util.Arrays;
-import java.util.List;
+import org.akv.two_pointers.easy.CountPairsWhoseSumIsThanTarget;
+import org.akv.two_pointers.easy.FindFirstPallindromicStringInArray;
+import org.akv.two_pointers.easy.PermutationsOfTheDiString;
+import org.akv.two_pointers.easy.SquaresOfASortedArray;
 
 public class Main {
```
Comment (PPapnai) [inline src/main/java/org/akv/Main.java:8]: no more main class in the code

File: src/main/java/org/akv/two_pointers/easy/FindFirstPallindromicStringInArray.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+//2108. Find First Palindromic String in the Array
+
+    public String firstPalindrome(String[] words) {
+
+        for(int i =0 ; i< words.length;i++){
+            if(checkPallindrome(words[i])) return words[i];
+        }
+        return "";
+    }
+    public boolean checkPallindrome(String word){
```
Comment (PPapnai) [inline src/main/java/org/akv/two_pointers/easy/FindFirstPallindromicStringInArray.java:14]: Method name: isPalindrome

**Pull request closed without merging**